### ISSUE TITLE
Possible to set an exact location, not just a city?

### ISSUE THREAD

**Issue by kyranb** [mfkp/tinderplusplus#46]
Any reason for only querying cities, would it be possible to geocode any location (to be more precise)?

**Comment by mfkp** [comment]
It's certainly possible, it's just set to cities right now to make it
easier for users to switch locations. Is there something that a more
precise location would get you?

**Comment by kyranb** [comment]
Absolutely, in large cities it's great to be able to be more specific and
just choose another area. It would take slightly more to implement, but a
popover map with pin would probably offer the best user experience
(replicating the mobile app).

Slightly off topic as well, would you consider adding an option to update
your profile's age and distance filters?

**Comment by mfkp** [comment]
Maybe it would be good enough to just be able to set an address in the search dropdown?

As for your other question, age and distance filters is definitely possible, just need to find the tinder API call for that and add it to the UI.
